src/icp_hello_world_rust_backend/tests/integration_tests.rs

```diff
@@ -9,7 +9,7 @@ fn setup() -> (PocketIc, Principal) {
 
     let backend_canister = pic.create_canister();
     pic.add_cycles(backend_canister, 2_000_000_000_000); // 2T Cycles
-    let wasm = fs::read(BACKEND_WASM).expect("Build the wasm with ./scripts/build.sh");
+    let wasm = fs::read(BACKEND_WASM).expect("Wasm file not found, run 'dfx deploy'.");
```

Comment:
Running dfx deploy might not be very convenient as it requires running dfx start first. But I don't have a better suggestion at hand either.

Reply:
True. dfx is started when the container is first opened, but it can stop later. The readme has a section that tells the users that they might have to run dfx start before dfx deploy.